include/riak_kv_ts.hrl

```diff
@@ -81,4 +81,9 @@
             'SHOW_TABLES' = <<>>  :: binary()
         }).
 
+-record(riak_sql_explain_query_v1,
```

Comment:
I forgot to mention it in the mumble, but I thought we wanted to do everything **one way**. The DDL has been macroed a while back, I have had to macro the sql_select record in group by, and param in descending keys cc @gordonguthrie